satpy/etc/readers/seviri_l2_grib.yaml

```diff
@@ -73,244 +73,274 @@ datasets:
     parameter_number: 39
     units: "1"
-    long_name: cloud_optical_depth_standard_error
 
   error_in_lower_layer_cloud_top_pressure:
     name: error_in_lower_layer_cloud_top_pressure
+    long_name: Lower Cloud Top Pressure Error Estimate
+    standard_name: air_pressure_at_cloud_top_in_lower_atmosphere_layer standard_error
     resolution: 3000.403165817
     file_type: grib_seviri_oca
     parameter_number: 40
     units: Pa
-    standard_name: air_pressure_at_cloud_top_standard_error
 
   fire_probability:
     name: fire_probability
+    long_name: Fire Probability
+    standard_name: fire_probability
     resolution: 3000.403165817
     file_type: grib_seviri_fir
     parameter_number: 192
     units: "%"
-    long_name: fire_probability
 
   active_fires:
     name: active_fires
+    long_name: Active Fire Classification
+    standard_name: active_fire_classification
     resolution: 3000.403165817
     file_type: grib_seviri_fir
     parameter_number: 9
     units: "1"
-    long_name: active_fire_classification
 
   aerosol_optical_thickness_vis06:
     name: aerosol_optical_thickness_vis06
+    long_name: Aerosol optical Thickness at 0.6um
+    standard_name: atmosphere_absorption_optical_thickness_due_to_ambient_aerosol
     resolution: 9001.209497451
     file_type: grib_seviri_aes
     parameter_number: 20
     units: "1"
-    long_name: aerosol_optical_thickness_vis06
 
   aerosol_optical_thickness_vis08:
     name: aerosol_optical_thickness_vis08
+    long_name: Aerosol optical Thickness at 0.8um
+    standard_name: atmosphere_absorption_optical_thickness_due_to_ambient_aerosol
     resolution: 9001.209497451
     file_type: grib_seviri_aes
     parameter_number: 21
     units: "1"
-    long_name: aerosol_optical_thickness_vis08
 
   aerosol_optical_thickness_vis16:
     name: aerosol_optical_thickness_vis16
+    long_name: Aerosol optical Thickness at 1.6um
+    standard_name: atmosphere_absorption_optical_thickness_due_to_ambient_aerosol
     resolution: 9001.209497451
     file_type: grib_seviri_aes
     parameter_number: 22
     units: "1"
-    long_name: aerosol_optical_thickness_vis16
 
   angstroem_coefficient:
     name: angstroem_coefficient
+    long_name: Angstroem Coefficient
+    standard_name: aerosol_angstrom_exponent
     resolution: 9001.209497451
     file_type: grib_seviri_aes
     parameter_number: 23
     units: "1"
-    long_name: angstroem_coefficient
 
   aes_quality:
     name: aes_quality
+    long_name: AES Product Quality Flag
+    standard_name: quality_flag
     resolution: 9001.209497451
     file_type: grib_seviri_aes
     parameter_number: 192
     units: "1"
-    long_name: aes_quality
 
   cloud_top_height:
     name: cloud_top_height
+    long_name: Cloud Top Height
+    standard_name: height_at_cloud_top
     resolution: 9001.209497451
     file_type: grib_seviri_cth
     parameter_number: 2
     units: m
-    long_name: cloud_top_height
 
   cloud_top_quality:
```

Comment:
Please add `flag_values` and `flag_meanings`.

Reply:
done